### DISCUSSION TITLE
Error : xhr poll error

### DISCUSSION THREAD

**Discussion by raaaju** [socketio/socket.io#3963] [Q&A]
Hello, I am using latest version of socket io client and could not connect to the socket.io server.
server to connect: "https://www.srplivehelp.com:2020",
also tried with rejectUnauthorized: false but no luck in connection

**Comment by darrachequesne** [comment]
Hi! Did you check the [troubleshooting guide](https://socket.io/docs/v4/troubleshooting-connection-issues/#Problem-the-socket-is-not-able-to-connect)?
